README.md

```diff
@@ -12,11 +12,13 @@ For a developer experience closer to the one the project maintainers current hav
 * [nvm](https://github.com/creationix/nvm)
 * Node.js and npm (use nvm to install them)
 * [Yarn](https://yarnpkg.com/)
-* [AndroidStudio](https://developer.android.com/studio/) to be able to compile the Android version of the app
+* [Android Studio](https://developer.android.com/studio/) to be able to compile the Android version of the app
 * [Xcode](https://developer.apple.com/xcode/) to be able to compile the iOS app
 * CocoaPods(`sudo gem install cocoapods`) needed to fetch React and third-party dependencies.
 * [Carthage](https://github.com/Carthage/Carthage#installing-carthage) for appium to be able run iOS UI tests
 
+There may be some configurations needed in Android Studio and Xcode, depending on your setup. Please refer to [React Native's documentation](https://reactnative.dev/docs/environment-setup) 
```

Comment:
```suggestion
There may be some configurations needed in Android Studio and Xcode, depending on your setup. Please refer to [React Native's documentation](https://reactnative.dev/docs/environment-setup).
```

Reply:
Thank you! Your review made me realise that the last part of the sentence, `for the latest requirements for each development environment.`,  was somehow missing. I've added that now!